usr.sbin/daemon/daemon.c

```diff
@@ -762,3 +755,19 @@ daemon_state_init(struct daemon_state *state)
 		.output_filename = NULL,
 	};
 }
+
+
+static _Noreturn void
+daemon_terminate(struct daemon_state *state)
+{
+	assert(state);
```

Comment:
Pointer should be explicitly compared against NULL when used like this

Reply:
ack, will fix in a bit

Reply:
fixed